5.5/run-mysqld-slave.sh

```diff
@@ -6,6 +6,7 @@ source ${HOME}/common.sh
 set -eu
 
 export MYSQL_RUNNING_AS_SLAVE=1
+export MYSQL_MASTER_ADDR=${MYSQL_MASTER_ADDR:-mysql-master}
```

Comment:
@bparees I think it is searching the current namespace for the "mysql-master" service. The variable is not provided, it is there so you can override it when you do bash testing with docker (where you don't have the headless service available)